libs/domain/cart/src/guards/cart-price.guard.ts

```diff
@@ -1,12 +1,12 @@
 import { Guard } from '@spryker-oryx/core';
-import { inject } from '@spryker-oryx/di';
+import { INJECTOR, inject } from '@spryker-oryx/di';
 import { Observable } from 'rxjs';
 import { CartService } from '../services';
 
 export class CartPriceModeChangeGuard implements Guard {
-  constructor(protected cartService = inject(CartService)) {}
+  constructor(protected injector = inject(INJECTOR)) {}
```

Comment:
Why set injector instead of cartService on first place ?

Reply:
we have circular then